### PR TITLE
hideButtons prop added

### PR DESCRIPTION
**Pull request by WandersonAlves**

### CODE REVIEW

**Comment by cristijora** [comment]
Thanks for the PR. So this hides only the back button ?

**Comment by WandersonAlves** [comment]
On my tests, this hides all buttons. The div class="wizard-card-footer" isn't the container for buttons? :thinking:

**Comment by WandersonAlves** [comment]
Imagine a table with a bunch of 'pending products' and i want to expand the row and see the status of the shipment, it's a perfect case for a read-only wizard step! 😃

File: src/components/FormWizard.vue [hunk 1]
```diff
@@ -45,7 +45,7 @@
       </div>
     </div>
 
-    <div class="wizard-card-footer">
+    <div class="wizard-card-footer" v-if="!hideButtons">
```
Comment (cristijora) [inline src/components/FormWizard.vue]: Let's reverse this to `v-if="hideButtons"
This way we keep it backwards compatible and the default behavior will display buttons
`:hide-buttons="true"` will actually hide them
Also it's more natural to say that `true` for `hide-buttons` actually hides buttons than the other way around

File: dev-example/App.vue [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
     <form-wizard @on-complete="onComplete"
+                 :hide-buttons="false"
```
Comment (cristijora) [inline dev-example/App.vue]: It's more natural to say that true for hide-buttons actually hides buttons than the other way around

**Comment by cristijora** [comment]
Merged it. Thanks for the response and the use case details and for the PR

**Pull request merged**